tests/pedal/test_pedal.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+import unittest
+from panda import Panda
+from panda_jungle import PandaJungle  # pylint: disable=import-error
+from canhandle import CanHandle
+
+JUNGLE_SERIAL = os.getenv("PEDAL_JUNGLE")
+
+class TestPedal(unittest.TestCase):
+  PEDAL_BUS = 1
+  def setUp(self):
+    self.jungle = PandaJungle(JUNGLE_SERIAL)
+    self.jungle.set_panda_power(True)
+    self.jungle.set_ignition(False)
+
+  def tearDown(self):
+    self.jungle.close()
+
+  def _flash_over_can(self, bus, fw_file):
+    print(f"Flashing {fw_file}")
+    while len(self.jungle.can_recv()) != 0:
+      continue
+    self.jungle.can_send(0x200, b"\xce\xfa\xad\xde\x1e\x0b\xb0\x0a", bus)
+
+    time.sleep(0.1)
+    with open(fw_file, "rb") as code:
+      PandaJungle.flash_static(CanHandle(self.jungle, bus), code.read())
+
+  def test_aaa_flash_over_can(self):
```

Comment:
split this up. first flash either fw and just check it comes up. then do a test for the USB flag